pgjdbc/src/main/java/org/postgresql/core/v3/ConnectionFactoryImpl.java

```diff
@@ -133,7 +138,8 @@ public QueryExecutor openConnectionImpl(HostSpec[] hostSpecs, String user, Strin
     Iterator<HostSpec> hostIter = hostChooser.iterator();
     while (hostIter.hasNext()) {
       HostSpec hostSpec = hostIter.next();
-      LOGGER.log(Level.FINE, "Trying to establish a protocol version 3 connection to {0}", hostSpec);
+      LOGGER.log(Level.FINE, "Trying to establish a protocol version 3 connection to {0}",
```

Comment:
>That autoformat caused all the changes to formatting that you were referring to before.

Well, you could confine autoformat to the selection.

Here's what I have:
<img width="1300" alt="import_order" src="https://user-images.githubusercontent.com/213894/27091245-3c893bbc-5068-11e7-8499-b04c83d1fe5a.png">
<img width="928" alt="import_order_rest" src="https://user-images.githubusercontent.com/213894/27091252-416a62aa-5068-11e7-9898-e38962e87ade.png">